modules/statics/src/coins.ts

```diff
@@ -2046,6 +2046,15 @@ export const coins = CoinMap.fromCoins([
     '0x4ddc2d193948926d02f9b1fe9e1daa0718270ed5',
     UnderlyingAsset.CETH
   ),
+  erc20(
+    'fd41106e-cf6f-4f0c-9868-a41fbcec8a49',
+    'cfx',
+    'Conflux',
+    18,
+    //Change
+    '0x4575f41308ec1483f3d399aa9a2826d74da13deb',
+    UnderlyingAsset.CFX
+  ),
```

Comment:
This is labeled as Orchid on Etherscan

Reply:
Fixed, currently using contract address found at bitgo-microservices/packages/lumina/packages/lumina-server-shared/resources/service/securitymaster/instrument/instrument_metadata.json